app/config/databaseConfig.js

```diff
@@ -7,7 +7,7 @@ function isProd () {
 
 const dbConfig = {
   username: process.env.POSTGRES_USER || 'postgres',
-  password: process.env.POSTGRES_PASSWORD || 'postgres',
+  password: process.env.POSTGRES_PASSWORD || 'ppp',
```

Comment:
Is this default password used for local postgresql instance?

Reply:
yes, it is only for local. just matching with the other scripts.